packages/flutter/test/material/checkbox_test.dart

```diff
@@ -1577,9 +1577,12 @@ void main() {
       )
     );
 
-    final Finder tooltip1 = find.byType(Tooltip);
+    await tester.pump(const Duration(days: 1));
```

Comment:
These tests were assuming the OverlayEntry tooltip updates when the `Tooltip` widget updates so I had to add code to wait for the previous tooltip to dismiss.